scripts/deposit.js

```diff
@@ -31,6 +32,14 @@ module.exports = async (callback) => {
     console.log("Preparing transaction data...")
     const transaction = await buildDepositFromList(masterSafe.address, deposits, true)
 
+    assert(
+      await isOnlyFleetOwner(
+        masterSafe.address,
+        deposits.map(({ bracketAddress }) => bracketAddress)
+      ),
+      "All depositors must be owned only by the master Safe"
+    )
```

Comment:
I actually wonder if this verification is even necessary. For example, what if I wanted to use an already existing safe as a bracket trader and simply configure it in a way that the `FundAccount` is its owner. I don't see that this should cause problems.

Reply:
Silly example: you, user, copypaste the example command from the readme:
```
npx truffle exec scripts/transfer_approve_deposit.js --masterSafe=$MASTER_SAFE --depositFile="./examples/exampleDepositList.json" --network=$NETWORK_NAME
```
You replaced `MASTER_SAFE` in your env, but forgot to change `depositFile` and run the script. You'd be (irrecoverably) depositing funds from your master safe to the example addresses. It's a user error, but something easily preventable by us, while the use case you describe is very specific. I think it's fair to have a user who has this particular need comment out the check manually.